room1/room1.txt

```diff
@@ -0,0 +1,10 @@
+Alex Arnaud 
+Cedric
+Peter
+Alfred
+Steve
+Abdoul
+Jen
```

Comment:
I think it's supposed to be Jen Foko instead of Jen. Want you to check it out, Alex

Reply:
Yes, Cedric, you are right. it is Jen Foko. Leader Alex thanks for taking a look

Reply:
@fjen53 Yeah that's what I ve been thinking. Alex, will you consider this feedback ?

Reply:
@s3alex What do you think ?